browser-checklist.md

```diff
@@ -0,0 +1,26 @@
+## Browsers
+
+- [ ] Chrome
+- [ ] Safari
+- [ ] FF
+- [ ] IE10
+- [ ] IE9
+- [ ] IE8
+- [ ] IE7
+- [ ] Opera
+- [ ] iOS8
+- [ ] Android 5
+- [ ] Blackberry Bold
+
+## Accessibility
+
+- [ ] Keyboard friendly
+- [ ] Screen reader friendly
+
+## Other
+
+- [ ] Is useable without CSS
+- [ ] Is useable without JS
+- [ ] Flexible from small to large screens
+- [ ] No linting errors or warnings
+- [ ] JavaScript tests are passing
```

Comment:
JS tests and CSS/JS linting are not part of "browser testing" per se. I think they're both better covered under something like #38.